run.py

```diff
@@ -0,0 +1,15 @@
+from livereload import Server, shell
```

Comment:
I know there's already a lot of clutter in the root folder. It's a bit late to ask that, but it would have been nice to make all the *tooling* parts live in a subfolder, e.g. `build-support`, so that the root folder really only holds the data for the site. Do you think it's the time we start doing that for the newly added tooling?

(This is somewhat attached to the `serve` alias, where it wouldn't matter where this script lives.)

Reply:
I also dislike this big root folder, but I woulnd't like to compilcate this PR.

Lets plan this for the new tooling or maybe even to do it together with #104.